test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -2383,6 +2383,9 @@ def convert_to_cli(self, attrs, op=None, hostname=None, dflt_conf=True,
             attrs = attrs.items()
 
         for a, v in attrs:
+            if a == "Job_Name":
```

Comment:
This adds an extra slash before every special character, not escape it, for centos8:
`sudo -H -u pbsuser /opt/pbs/bin/qsub -N j\&whoami\&gt\;\/tmp\/b\& -- /bin/sleep 100`
So, i think you need to make this platform specific for only the platforms which have this issue